roadmap/README.md

```diff
@@ -0,0 +1,148 @@
+# Tekton Mission and Roadmap
+
+This doc describes Tekton's mission and the 2020 roadmap primarily
```

Comment:
This doc should only be on Tekton's mission and main roadmap (aka as "what we want our components" to lead to) and have a ROADMAP document for the each project on their own.

Reply:
Question is then, should the ROADMAP for the other project be here or on their own porject ?

Reply:
ah kk! as long as we link from here maybe that's the way to go. i suppose on tekton.dev we can (hopefully) bring it all together!